app/controllers/pieces_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   def update
     @piece = Piece.find(params[:id])
     @game = Game.find(@piece.game_id)
-    @piece.update_attributes(piece_params)
+    @piece.move_to!(piece_params[:row_position].to_i, piece_params[:col_position].to_i)
```

Comment:
You're right, @addstar34.  I thought that in this case, since the user is not given the opportunity to pass any data and the row and column information comes from the way we generate the board, there would be no danger.  That said, I'm not sure of any of that.  Perhaps it would be safest to stick with strong params.  Are we using the `:type` key anywhere, though?  Maybe that could go.